src/pages/authorization/authorizing-api-calls/index.mdx

```diff
@@ -793,6 +793,16 @@ To achieve that, make the same exact same request as in [Case: New customer](#ca
 
 Your `entity_id`, or any other customer identity data, won' be overwritten. The web browser will attach cookies to the server allowing it to recognize you as an existing customer. You'll get a new access token, and cookies' lifetime will be extended, too. They will be valid for 2 years from your last request to `https://accounts.livechat.com/customer/token`.
 
+
+### Case: Existing customer chatting in different groups
+
+Use this option to generate a new access token for an already existing customer, but with separate identity for another group (this requires `chat_between_groups` license property to be set to `false`).
```

Comment:
```suggestion
Use this option to generate a new access token for an already existing customer, but with a separate identity for another group. (This requires the `chat_between_groups` license property to be set to `false`.)
```

Reply:
+ link to the endpoint?

Reply:
Sorry for the lack of clarity. I meant linking to the `{organization_id}/{group_id}/token/` endpoint in the Customer Accounts API spec. But we can do that once the spec is rendered – I don't know the shape of the link at this stage.